misk/src/main/kotlin/misk/web/DashboardTab.kt

```diff
@@ -8,47 +8,89 @@ import kotlin.reflect.KClass
 class DashboardTab(
```

Comment:
any reason these aren't data classes?

Reply:
It doesn't seem to work with the way it extends WebTab class which also extends ValidWebEntry

Reply:
gotcha